tests/fixtures/mod.rs

```diff
@@ -223,3 +227,51 @@ pub fn tempfile() -> std::fs::File {
 pub fn duckdb_conn() -> duckdb::Connection {
     duckdb::Connection::open_in_memory().unwrap()
 }
+
+#[allow(dead_code)]
```

Comment:
Does the dead code attribute need to be here?

Reply:
Without it the clippy lint check fails